Makefile

```diff
@@ -98,7 +98,7 @@ coverage-cpp:
 test-python: test-builtin test-suite
 
 test-builtin:
-	PL_DEVICE=$(if $(device:-=),$(device),lightning.qubit) $(PYTHON) -I $(TESTRUNNER)
+	PL_DEVICE=$(if $(device:-=),$(device),lightning.qubit) $(PYTHON) $(TESTRUNNER)
```

Comment:
The update I asked for in the Makefile is not this one. You can keep that.
We need to update the call for editable installation everywhere, including here.
This is all that is needed.